jetty-ee9/jetty-ee9-tests/jetty-ee9-test-client-transports/src/test/java/org/eclipse/jetty/ee9/test/client/transport/AsyncIOServletTest.java

```diff
@@ -100,14 +93,52 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.fail;
 
-public class AsyncIOServletTest extends AbstractTest<AsyncIOServletTest.AsyncTransportScenario>
+// TODO: most of these tests do not work because the scope listener mechanism is broken.
+@Disabled
```

Comment:
Is this handled by the listener PR from @lorban ?

Reply:
No, these are server-side listeners.